drivers/counter/counter_mcux_rtc.c

```diff
@@ -236,6 +236,14 @@ static int mcux_rtc_init(const struct device *dev)
 	RTC_GetDefaultConfig(&rtc_config);
 	RTC_Init(config->base, &rtc_config);
 
+#ifdef CONFIG_SEL_LPO_1KHZ_CLOCK
+	/* The RTC prescaler increments using the LPO 1 kHz clock intead of RTC 32.768 kHz clock.
```

Comment:
```suggestion
	/* The RTC prescaler increments using the LPO 1 kHz clock instead of the RTC clock */
```
avoid adding board related details in this generic driver. Also not all devices have an internal RTC of 32.768 Khz, so refer to it only as RTC clock.

Reply:
Updated.